src/features/SelectGroup/SelectGroup.tsx

```diff
@@ -30,6 +31,8 @@ export default function SelectGroup({
     }
   };
 
+  const selectStyle = `${Style.selectGroup__select} selectGroup`;
```

Comment:
You can just put `.selectGroup` class on `<div className={Style.selectGroup}>` instead of putting this class on three Selects